Specs/DateTime/Dutch/DatePeriodExtractor.json

```diff
@@ -3075,14 +3075,14 @@
     ]
   },
   {
-    "Input": "Het zal 2 dagen vanaf nu gebeuren",
```

Comment:
@LionbridgeCSII @tellarin this is the only case I am not certain about. In the English specs this is mentioned as `from now`, which would be in Dutch:

'vanaf nu' -> 'from now'
'vanaf vandaag' -> 'from today'

The current change with 'daarna' (afterwards, after that) is different. And for me, the issue would be that it is unclear to what it refers to. The original form mentions that explicit, where 'daarna' could be based on a previous sentence..

Reply:
For '2 days hence' I can understand this suggestion! 

https://github.com/microsoft/Recognizers-Text/blob/51ee58ab19624a6bf431fcdccc3779743b506d0f/Specs/DateTime/English/MergedParser.json#L3324

Perhaps it would make sense to add these cases, even if they are unsupported for now? Currently in English and Dutch only "3 minuten vanaf nu" is used, however this is not a test cases for days.

It would make sense to me to have this added as a testcase;
"Het evenement zal 2 dagen vanaf nu plaatsvinden."

Reply:
ok, I will add this test case.